app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -104,31 +106,27 @@ export default function CompetingStep({
 
   const hasChanges = hasEventsChanged || hasCommentChanged || hasGuestsChanged;
 
-  const commentIsValid = comment.trim() || !competitionInfo.force_comment_in_registration;
```

Comment:
Is this being checked somewhere else?

Reply:
yes, this is now checked by the native form element by setting the comment to required and the button to type submit